themes/codeMirror/customEditorStyles.less

```diff
@@ -0,0 +1,68 @@
+.editor .codeEditor .CodeMirror {
+    // Themes with dark backgrounds
+    &.cm-s-3024-night,
+    &.cm-s-abbott,
+    &.cm-s-abcdef,
+    &.cm-s-ambiance,
+    &.cm-s-ayu-dark,
+    &.cm-s-ayu-mirage,
+    &.cm-s-base16-dark,
+    &.cm-s-bespin,
+    &.cm-s-blackboard,
+    &.cm-s-cobalt,
+    &.cm-s-colorforth,
+    &.cm-s-darcula,
+    &.cm-s-dracula,
+    &.cm-s-duotone-dark,
+    &.cm-s-erlang-dark,
+    &.cm-s-gruvbox-dark,
+    &.cm-s-hopscotch,
+    &.cm-s-icecoder,
+    &.cm-s-isotope,
+    &.cm-s-lesser-dark,
+    &.cm-s-liquibyte,
+    &.cm-s-lucario,
+    &.cm-s-material,
+    &.cm-s-material-darker,
+    &.cm-s-material-ocean,
+    &.cm-s-material-palenight,
+    &.cm-s-mbo,
+    &.cm-s-midnight,
+    &.cm-s-monokai,
+    &.cm-s-moxer,
+    &.cm-s-night,
+    &.cm-s-nord,
+    &.cm-s-oceanic-next,
+    &.cm-s-panda-syntax,
+    &.cm-s-paraiso-dark,
+    &.cm-s-pastel-on-dark,
+    &.cm-s-railscasts,
+    &.cm-s-rubyblue,
+    &.cm-s-seti,
+    &.cm-s-shadowfox,
+    &.cm-s-the-matrix,
+    &.cm-s-tomorrow-night-bright,
+    &.cm-s-tomorrow-night-eighties,
+    &.cm-s-twilight,
+    &.cm-s-vibrant-ink,
+    &.cm-s-xq-dark,
+    &.cm-s-yonce,
+    &.cm-s-zenburn
```

Comment:
Ah, nice way to recolor our custom items.

In a future PR, would it be possible to move the default coloring for "light" styles in here as well? Just to keep similar items in the same place?

Reply:
I've added a section to the LESS for the CM themes with light backgrounds - it's a much shorter list. As the Linter will complain about an empty style, the styling currently sets the CSS variable `--dummyVar` to a value of `'currently unused'`.